include/controller/cpp/webots/Supervisor.hpp

```diff
@@ -59,6 +60,8 @@ namespace webots {
     Node *getSelf() const;
     Node *getFromDef(const std::string &name) const;
     Node *getFromId(int id) const;
+    Node *getFromDevice(const Device *device) const;
+    Node *getFromDeviceTag(int tag) const;
```

Comment:
No, we should not document `getFromDeviceTag`. because we don't want that users call them given that 
This is public just because we need to use it in the Java API.
But this function is private in all the other APIs.
Note that `Device.getTag` is not documented given that is public in C++ just because it is used in other APIs.